alacritty_terminal/src/cli.rs

```diff
@@ -60,8 +60,11 @@ impl Options {
     pub fn load() -> Options {
         let mut options = Options::default();
 
+        let git_sha = env!("GIT_SHA");
+        let version_string = format!("{} ({})", crate_version!(), git_sha);
```

Comment:
I think the solution here might be to move this into the `alacritty` crate itself. It seems like a better home for this.

Reply:
Feel free to go ahead with this. It's definitely not going to stay in `alacritty_terminal` in the long run. So moving it to `alacritty` sounds good to me.